app/components/history_version_component.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+# UI History Version Display Component
+class HistoryVersionComponent < Component
+  include JsonHelper
+
+  attr_accessor :log_data
+
+  def initialize(
+    log_data: nil,
+    **system_arguments
```

Comment:
**nitpick:** Again here, probably don't need system_arguments

Reply:
Removed in 2a1c80ebd2a76e36237c5e4ad897c612d31278e1